tools.go

```diff
@@ -6,18 +6,19 @@
 package tools
 
 import (
-	// Makefile
 	"github.com/jteeuwen/go-bindata/go-bindata"
-	"github.com/openshift/library-go/alpha-build-machinery/make/targets/openshift"
+	// Makefile
```

Comment:
this one is better to keep as is

Reply:
Fixed, good catch.  I need to update the script to preserve comments, or allow an exclude flag